### PR TITLE
Add inject_tlm to javascript api

### LINKED ISSUE #436: Expose inject_tlm to the javascript api allowed commands.
Hello,

I control a couple microservices via packets (by injecting tlm) and was wondering if the inject_tlm api call could be included as an option for a screen, as I would like to control a micorservice from a screen rather than kicking off a script.

image

Thanks!

Dakota

### PR DESCRIPTION
**Pull request by ryanmelt**
closes #436

### CODE REVIEW

File: openc3-cosmos-init/plugins/packages/openc3-tool-common/src/services/openc3-api.js [hunk 1]
```diff
@@ -335,7 +335,7 @@ export class OpenC3Api {
       }
```
Comment (jmthomas) [inline openc3-cosmos-init/plugins/packages/openc3-tool-common/src/services/openc3-api.js:335]: Line 333 has a bug: `Invalid data type ${packet_name}` ... it should be `${value_type}`.

File: openc3-cosmos-init/plugins/packages/openc3-tool-common/src/services/openc3-api.js [hunk 2]
```diff
@@ -335,7 +335,7 @@ export class OpenC3Api {
...
     } else {
       data = await this.exec('tlm', [target_name, packet_name, item_name], {
-        type: data_type,
+        type: value_type,
```
Comment (jmthomas) [inline openc3-cosmos-init/plugins/packages/openc3-tool-common/src/services/openc3-api.js:338]: We're not at all consistent with the term `data_type` vs `value_type` throughout the code. I assume you're indicating you like value_type going forward.

File: openc3/lib/openc3/api/tlm_api.rb [hunk 3]
```diff
@@ -118,7 +118,8 @@ def set_tlm(*args, type: :CONVERTED, scope: $openc3_scope, token: $openc3_token)
     # @param type [Symbol] Telemetry type, :RAW, :CONVERTED (default), :FORMATTED, or :WITH_UNITS
     def inject_tlm(target_name, packet_name, item_hash = nil, type: :CONVERTED, scope: $openc3_scope, token: $openc3_token)
       authorize(permission: 'tlm_set', target_name: target_name, packet_name: packet_name, scope: scope, token: token)
-      unless CvtModel::VALUE_TYPES.include?(type.intern)
+      type = type.to_s.intern
+      unless CvtModel::VALUE_TYPES.include?(type)
```
Comment (jmthomas) [inline openc3/lib/openc3/api/tlm_api.rb:122]: Why this change? You can call `intern` on an existing symbol ... it's a noop.

**Review by jmthomas — CHANGES_REQUESTED** [review]
Please fix the one existing typo

File: openc3/lib/openc3/api/tlm_api.rb (hunk 3 above)
Comment (ryanmelt) [reply openc3/lib/openc3/api/tlm_api.rb:122]: handles nil and strings coming in (which is what was happening in inject_tlm)

File: openc3-cosmos-init/plugins/packages/openc3-tool-common/src/services/openc3-api.js (hunk 1 above)
Comment (ryanmelt) [reply openc3-cosmos-init/plugins/packages/openc3-tool-common/src/services/openc3-api.js:335]: This isn't a bug.  packet_name is the second positional argument, and this case is for the two argument case.

**Review by jmthomas — APPROVED** [review]
